source/fuzz/transformation_swap_two_functions.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2021 Shiyu Liu 
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "source/fuzz/transformation_swap_two_functions.h"
+
+#include "source/opt/function.h"
+#include "source/opt/module.h"
+#include <iostream>
+#include <string>
+
+namespace spvtools {
+namespace fuzz {
+
+TransformationSwapTwoFunctions::TransformationSwapTwoFunctions(
+    protobufs::TransformationSwapTwoFunctions message)
+    : message_(std::move(message)) {}
+
+TransformationSwapTwoFunctions::TransformationSwapTwoFunctions(uint32_t id1, uint32_t id2) {
+  message_.set_function_id1(id1);
+  message_.set_function_id2(id2);
```

Comment:
> You could assert that the function ids cannot be the same here.

If I assert that id cannot be the same here, wouldn't it fail the test because `assert` will abort when it fails?

Reply:
Yes, the line that contains `TransformationSwapTwoFunctions(4, 4)` in your test would crash, but you could use ASSERT_DEATH or remove that line.

Reply:
Fixed.